### ISSUE TITLE
Activated environment with windows and bash

### ISSUE THREAD

**Issue by galen104** [goanpeca/setup-miniconda#5]
I'm doing some experiments with `setup-miniconda`, but I have a problem with the following code:

```
steps:
      - name: Checkout Repo
        uses: actions/checkout@master
      - uses: goanpeca/setup-miniconda@master
        with:
          auto-update-conda: false
          python-version: ${{ matrix.python-version }}  
          activate-environment: test-environment
          environment-file: environment.yml
          condarc-file: condarc.yml
          auto-activate-base: false
      - name: Flake8
        shell: bash -l {0}
        run: |
           conda env list
           flake8 src tests
```

`environment.yml` and `condarc.yml` are ok, all dependencies are installed and the environment `test-environment` is created correctly, but in the next step (Flake8), the activated environment is base and not `test-environment` if the os is windows and the shell is bash or sh. For example with linux/bash or windows/pwsh all works as expected. 
If can be useful, this is `.bash_profile` on windows:

```
Run cat $HOME/.bash_profile

# >>> conda initialize >>>
# !! Contents within this block are managed by 'conda init' !!
eval "$('/C/Miniconda/Scripts/conda.exe' 'shell.bash' 'hook')"
# <<< conda initialize <<<
```

As a quick hack, if I add the following step:

```
   - name: Activate `test-environment` (Windows)
        if: runner.os == 'Windows'
        run: echo "activate test-environment" >> ~/.bash_profile
```

the Flake8 step works.
Is that a normal behaviour? Maybe a bug?

Thanks.

**Comment by goanpeca** [comment]
Hello @galen104 . Could you share with me the repo or the contents of condarc and env.yaml?

Thanks!

**Comment by goanpeca** [comment]
Ok I think it is a bug. Let me make a new release to correct it ;-)

**Comment by goanpeca** [comment]
This has been fixed on v1.0.2 (Released already and v1 points to that release, so you can keep using v1 as Github docs suggest that)
